apps/portals/b2ai.standards/src/components/D4D.tsx

```diff
@@ -1,20 +1,55 @@
-import { GCInfo } from '@/config/GrandChallengeResources'
+import { ErrorBanner } from 'synapse-react-client'
+import { QueryBundleRequest } from '@sage-bionetworks/synapse-types'
+import useGetQueryResultBundle from 'synapse-react-client/synapse-queries/entity/useGetQueryResultBundle'
+import {
+  D4D_CONTENT_COLUMN_CONSTS,
+  getColumnExpressions,
+} from '@/config/resources'
+import {
+  getQueryBundleRequestWithIdFilter,
+  getRowsAsObjects,
+} from '@/hooks/fetchDataUtils'
 
 type D4DProps = {
   org_id: string
 }
 
 export function D4D(props: D4DProps) {
   const { org_id } = props
-  const html = GCInfo[org_id]?.d4dHtml ?? ''
 
-  if (!html) {
-    return <div>No datasheet available for organization: {org_id}</div>
-  }
+  const colExpressions = getColumnExpressions({ tableName: 'D4D_content' })
+
+  const queryBundleRequest: QueryBundleRequest =
+    getQueryBundleRequestWithIdFilter('D4D_content', colExpressions, [org_id])
+
+  const {
+    data = [],
+    error,
+    isLoading,
+  } = useGetQueryResultBundle(queryBundleRequest, {
+    select: result => {
+      return getRowsAsObjects(result)
+    },
+  })
 
+  if (error) {
+    return <ErrorBanner error={error} />
+  }
+  if (isLoading) {
+    return <div>Loading...</div>
```

Comment:
Consider a more user-friendly loader, such as `LinearProgress` from `@mui/material`
```suggestion
    return <LinearProgress />
```